Tests/Admin/ParkingSlotTests/Services/ParkingSlotServiceTests.cs

```diff
@@ -166,5 +166,21 @@ public void GivenParkingZoneIdAndCategoryAndIsSlotFree_WhenFilterByCategoryIsCal
             Assert.NotNull(result);
             Assert.IsAssignableFrom<IQueryable<ParkingSlot>>(result);
         }
+
+        [Fact]
+        public void GivenParkingZoneId_WhenGetAllReservationsByParkingZoneIdIsCalled_ThenRepositoryGetAllIsCalledAndReturnsReservations()
```

Comment:
```suggestion
        public void GivenParkingZoneId_WhenGetAllReservationsByParkingZoneIdIsCalled_ThenReturnsReservations()
```
why are you mentioning extra info we never named like this, never mentioned repository in other unit tests